model/src/main/kotlin/Identifier.kt

```diff
@@ -62,6 +64,37 @@ data class Identifier(
             name = "",
             version = ""
         )
+
```

Comment:
This code doesn't seem to be used outside of newly added tests and probably can be removed.